### ISSUE TITLE
Deluge error

### ISSUE THREAD

**Issue by bokiroki** [MichaIng/DietPi#1658]
#### Required Information:
- DietPi Version  -  v6.5  
- SBC Device  -  RPi 3 Model B (armv7l)
- Power supply used  -  official raspberry pi power supply
- SD card used  -  samsung evo+ 32 gb,    user data is on external hard drive

Dietpi is installed yesterday, with samba, deluge, sonarr, radarr, plex..

i got this failed message on Deluge few times after install... for example i just exited dietpi-drive_manager and got this message..
![untitled](https://user-images.githubusercontent.com/19166918/38094480-26b456a2-336e-11e8-8d5f-04cb084a203d.png)

Also, i have one question.. why i must log in to dietpi samba every time i power on my pc.. i click on remember my credentials and after restart my credentials are not there.. Thanks..
![untitled222](https://user-images.githubusercontent.com/19166918/38094739-f03f3fe6-336e-11e8-8960-c98c4d52eabc.png)

**Comment by Fourdee** [comment]
@bokiroki 

Thanks for the report 👍 

> i got this failed message on Deluge few times after install... for example i just exited dietpi-drive_manager and got this message..

Could just be the process has been swapped with another during the time we apply the tool. Or, the process does not support change. I'll try to replicate here.


> Also, i have one question.. why i must log in to dietpi samba every time i power on my pc.. i click on remember my credentials and after restart my credentials are not there.. Thanks..

Most likely a Windows specific issue (https://community.spiceworks.com/topic/1404239-windows-10-not-remembering-shared-network-drive-login-credentials).

Try mapping the drive: https://www.techrepublic.com/article/how-to-connect-to-linux-samba-shares-from-windows-10/

**Comment by Fourdee** [comment]
Confirmed. Although process appears to be running, most likely a swap in processes during launch.
Second run of ```dietpi-process_tool 1``` is fine.

**Comment by Fourdee** [comment]
🈯️ Resolved for v6.6 (and patched) using systemD forking process (which waits for the fork to complete):
```
root@DietPi:~# dietpi-services start
[  OK  ] Root access verified.

 DietPi-Services
─────────────────────────────────────────────────────
 Mode: start

[  OK  ] DietPi-Services | start : deluged
[  OK  ] DietPi-Services | start : deluge-web
[  OK  ] DietPi-Services | start : cron
[ SUB1 ] DietPi-Process_tool > Apply
[  OK  ] DietPi-Process_tool | Cron (6386) : Nice      0
[  OK  ] DietPi-Process_tool | Cron (6386) : Affinity  0-3
[  OK  ] DietPi-Process_tool | Cron (6386) : Scheduler SCHED_OTHER 0
[  OK  ] DietPi-Process_tool | Deluge (Daemon) (6364) : Nice      0
[  OK  ] DietPi-Process_tool | Deluge (Daemon) (6364) : Affinity  0-3
[  OK  ] DietPi-Process_tool | Deluge (Daemon) (6364) : Scheduler SCHED_OTHER 0
[  OK  ] DietPi-Process_tool | Deluge (Web interface) (6375) : Nice      0
[  OK  ] DietPi-Process_tool | Deluge (Web interface) (6375) : Affinity  0-3
[  OK  ] DietPi-Process_tool | Deluge (Web interface) (6375) : Scheduler SCHED_OTHER 0
```
